File: java/1-TwoSum.java
// 2022-3-29 17:44
// Runtime 50ms(34.5%)
// Memory 41.2mb(67.8%)

class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] ret = new int[2];
        for(int i = 0; i < nums.length; i++) {
            for(int j = i + 1; j < nums.length; j++) {
                if(nums[i] + nums[j] == target) {
                    ret[0] = i;
                    ret[1] = j;
                }
            }
        }
        return ret;
    }
}

File: cpp/1-TwoSum.cpp
// 2022-3-29 20:16
// Runtime 300ms(30.1%)
// Memory 9.7mb(98.1%)

class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> ret;
        bool find = false;
        for(int i = 0; i < nums.size(); i++) {
            if(find) break;
            for(int j = i + 1; j < nums.size(); j++) {
                if(nums[i] + nums[j] == target) {
                    ret = {i, j};
                    find = true;
                    break;
                }
            }
        }
        return ret;
    }
};
